book/graphics.md

```diff
@@ -365,7 +367,7 @@ Scrolling introduces a layer of indirection between page coordinates
 coordinates (since you've scrolled 60 pixels down, this text is 72
 pixels from the top of the *screen*)---see Figure 5. Generally speaking, a browser
 *lays out* the page---determines where everything on the page
-goes---in terms of page coordinates and then *renders* the
+goes---in terms of page coordinates and then *rasters* the
```

Comment:
This is arguable but in the rest of the book we definitely use "render" to refer to the whole pipeline including styling, layout, and painting, while "raster" refers to actual pixel drawing.